.travis.yml

```diff
@@ -56,6 +64,22 @@ before_install:
         phpenv config-rm xdebug.ini
       fi
 
+  - pecl channel-update pecl.php.net
+  - if [[ $CODECOVERAGE == 1 ]]; then pecl install pcov; fi
+
+  - |
+      if [ $DB = 'sqlserver' ]; then
+        sudo apt-get install -y mssql-server
+        sudo MSSQL_SA_PASSWORD="Strong!Passw0rd" MSSQL_PID=Express /opt/mssql/bin/mssql-conf -n setup accept-eula
+        sudo ACCEPT_EULA=Y apt-get install -y mssql-tools msodbcsql17 unixodbc-dev
+        pecl install sqlsrv
+        pecl install pdo_sqlsrv
+      fi
+
+  - if [ $DB = 'sqlserver' ]; then /opt/mssql-tools/bin/sqlcmd -b -S localhost -U SA -P "Strong!Passw0rd" -Q "CREATE DATABASE cakephp_test;"; fi
+  - if [ $DB = 'sqlserver' ]; then /opt/mssql-tools/bin/sqlcmd -b -S localhost -U SA -P "Strong!Passw0rd" -Q "CREATE DATABASE cakephp_test2;"; fi
+  - if [ $DB = 'sqlserver' ]; then /opt/mssql-tools/bin/sqlcmd -b -S localhost -U SA -P "Strong!Passw0rd" -Q "CREATE DATABASE cakephp_test3;"; fi
```

Comment:
I'm not sure we need all these databases. We don't use any beyond the first in any database server :blush:

Reply:
Let's give it a shot, searching shows none of them are being used afaik.